content/tutorials/manage-the-rippled-server/troubleshooting/health-check-interventions.md

```diff
@@ -0,0 +1,104 @@
+
+## Special Cases
+
+Certain server configurations may always report a `warning` status even when operating normally. If your server qualifies as a special case, you must configure your automated monitoring to recognize the difference between the normal status and an actual problem. This probably involves parsing the JSON response body for the health check method and comparing the values there with expected normal ranges.
+
+Some examples of special cases that may occur include:
+
+- A [private peer](peer-protocol.html#private-peers) typically has a very small number of peer-to-peer connections to known servers only, but the health check reports a warning on the `peers` metric if the server is connected to 7 or fewer peers. You should know the exact number of peers your server is configured to have and check for that value.
+- On a [parallel or test network](parallel-networks.html) where new transactions are not being sent continuously, the network waits up to 20 seconds for new transactions before attempting to validate a new ledger version, but the health check reports a warning on the `validated_ledger` metric if the latest validated ledger is 7 or more seconds old. If you are running `rippled` on a non-production network, you may want to ignore `warning` messages for this metric unless you know that there should be transactions being regularly sent. You may still want to alert on the `critical` level of 20 seconds, because the XRP Ledger protocol is designed to validate new ledger versions at least once every 20 seconds even if there are no new transactions to process.
+
+## Suggested Interventions
+
+When a health check fails, and it's not just a [momentary failure](#momentary-failures), the action to take to recover from the outage varies based on the cause. You may be able to configure your infrastructure to fix some types of failures automatically. Other failures require the intervention of a human administrator who can investigate and take the necessary steps to resolve more complex or critical failures; depending on the structure of your organization, you may have different levels of human administrator so that less skilled, lower level administrators can fix certain issues independently, but need to escalate to higher level administrators to fix larger or more complex issues. How and when you respond is likely to depend on your unique situation, but the metrics reported in the health check result can be a factor in these decisions.
+
+The following sections suggest some common interventions you may want to attempt and the health check statuses most likely to prompt those interventions. Automated systems and human administrators may selectively escalate through these and other interventions:
+
+- [Redirect traffic](#redirect-traffic) away from the affected server
```

Comment:
how are you organizing the list of interventions? right now, if i were reading this doc to troubleshoot my issue, i would think that Upgrade is the "last resort" intervention because it's listed last, but i don't think that's the case. might be helpful to have more of a troubleshooting flow here

Reply:
It was supposed to be in order of least to most severe interventions, but you're right that upgrading is kind of in a weird place there. That's because this is kind of a special case that doesn't appear elsewhere. But, I guess more generally, it's not a bad idea to upgrade. I'll move it up before the "investigate network" step.